sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -780,7 +780,7 @@ class SparkSession private(
     DataSource.lookupDataSource(runner, sessionState.conf) match {
```

Comment:
Actually @cloud-fan that would not work .. E.g., if PythonDataSource implements `ExternalCommandRunner`, we should load it here.

Reply:
lemme fix it separately. Reading the code path, I think it won't more and less affect.